E2E/DURABLE-FUNC-ORDER-PROCESSING/README.md

```diff
@@ -0,0 +1,125 @@
++ Start Azurite storage emulator. See [this page](https://learn.microsoft.com/azure/storage/common/storage-use-azurite) for how to configure and start the Azurite emulator for Local Storage.
++ Create a file named `local.settings.json` in **http** directory and add the following:
+
+  ```json
+  {
+    "IsEncrypted": false,
+    "Values": {
+      "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+      "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated"
+    }
+  }
+  ```
+
+### Using Azure Functions Core Tools (CLI)
+
+1) Open a new terminal and do the following:
+
+```bash
+cd DURABLE-FUNC-ORDER-PROCESSING
+func start
+```
+
+2) This sample uses an HTTP trigger to start an orchestration, so open a browser and go to http://localhost:7071/api/OrderProcessingOrchestration_HttpStart.
+
+![Screenshot of starting an orchestration instance through http](./img/start-orchestration-instance-http.png)
+
+3) To check the status of the orchestration instance started, go to the `statusQueryGetUri`. Your orchestration instance should show status "Running". After a few seconds, refresh to see that the orchestration instance is "Completed" and what the output is.
+
+![Screenshot of orchestration instance status](./img/orchestration-instance-status.png)
+
+### Using Visual Studio
+
+1) Open project using Visual Studio 2022 or later.
+2) Press Run/F5 to run in the debugger
+3) Use same approach above to start an orchestration instance and check its status. 
+
+### Using Visual Studio Code
+
+1) Open this folder in a new terminal
+2) Open VS Code by entering `code .` in the terminal
+3) Add a **.vscode** folder by running *"Azure Functions: Initialize project for use with VS Code"* in the Command Pallete
+4) Press Run/Debug (F5) to run in the debugger
+5) Use same approach above to start an orchestration instance and check its status. 
+
+
+## Provision the solution on Azure
+
+To set up this sample, follow these steps:
+
+1. Clone this repository to your local machine.
+2. in the root folder use the [Azure Developer CLI (azd)](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd?tabs=winget-windows%2Cbrew-mac%2Cscript-linux&pivots=os-windows) to provision a new resource group with the environment name you provide and all the resources for the sample.
+
+```bash
+azd up
+``` 
+
+3. Set always ready instance to reduce cold start latency. The Flex Consumption plan includes this feature to let you specify a number of instances that are always running to handle requests with minimal latency. By default, this is set to 0 but can be configured in [Azure Portal](https://learn.microsoft.com/azure/azure-functions/flex-consumption-how-to?tabs=azure-portal%2Cvs-code-publish&pivots=programming-language-csharp#set-always-ready-instance-counts) or with Azure CLI. We recommend that you set always ready instance to at least 1 for Durable Functions.
```

Comment:
Given we are provisioning the infra using bicep, we could already do the always ready configuration in the infra code and just mention that always ready was set to 1 for this sample to reduce cold start. [Here's an example](https://github.com/Azure/azure-rest-api-specs/blob/a3d10ee17c8f7ca4c2b4402862a1dc78fbab5c2a/specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/examples/CreateOrUpdateFunctionAppFlexConsumptionWithDetails.json#L38-L44) of how Always Ready config can be done in bicep. Note that this might raise questions as to why always ready is even needed, so suggest adding that note in the Flex Consumption Observations docs section that we discussed.

Reply:
good idea. Created a PR for the call out: https://github.com/MicrosoftDocs/azure-docs-pr/pull/286586